README.md

```diff
@@ -189,7 +189,7 @@ writeBambuOutput(se, path = "./bambu/")
 ```
 If you are only interested in the novel transcripts, one can filter this 'se' object first to remove reference annotations.
 ```rscript
-se.novel = se[mcols(se)$newTxClass != "annotation",]
+se.novel = se[mcols(se)$txClassDescription != "annotation",]
```

Comment:
we could add a line here to refelct the new columns that are exactly included for this purpose?